input/groups/back.json

```diff
@@ -4,78 +4,85 @@
       "Starter": [
         2340
       ],
-      "Crafting: Armorsmith": [
+      "Crafting Discipline: Armorsmith": [
         2304,
         2305,
         2308,
         2279,
         2301,
         2303
       ],
-      "Crafting: Leatherworker": [
+      "Crafting Discipline: Leatherworker": [
         2292,
         2293,
         2294,
         2295,
         2296,
         2315
       ],
-      "Crafting: Tailor": [
+      "Crafting Discipline: Tailor": [
         2307,
         2309,
         2282,
         2283,
         2316,
         2286
       ],
-      "Crafting: Artificer": [
+      "Crafting Discipline: Artificer": [
         2320,
         2297,
         2298,
         2299,
         2284,
         2300
       ],
-      "Crafting: Huntsman": [
+      "Crafting Discipline: Huntsman": [
         2321,
         2322,
         2323,
         2310,
         2311,
         2312
       ],
-      "Crafting: Weaponsmith": [
+      "Crafting Discipline: Weaponsmith": [
         2288,
         2290,
         2306,
         2291,
         2317,
         2287
       ],
-      "Crafting: Chef": [
+      "Crafting Discipline: Chef": [
         2289,
         2324,
         2277,
         2314,
         2302,
         2319
       ],
-      "Crafting: Jeweler": [
+      "Crafting Discipline: Jeweler": [
         2276,
         2278,
         2280,
         2281,
         2313,
         2285
       ],
-      "Crafting: Scribe": [
+      "Crafting Discipline: Scribe": [
         6512,
         6402,
         6341,
         6503,
         6361,
         6495
       ],
+      "Ascended": [
```

Comment:
I'ld be inclined to leave these in Standalone: Other unless we use something other than rarity to link them (as is it isn't clear why the Bough of Melandru wouldn't be grouped with them). They are all mystic forgable... I assume these were the original ascended back pieces as well so there might be something there.

Reply:
Four of them (the books) have the same exact skin, and all five of them come from the original Mystic Forge recipes for the first ascended back items they released, so I'd keep the group around. Bough of Melandru is a classic crafting recipe, nothing to do with them.

You could rename them into "Book / Quiver" or something like that to represent what's actually inside. I don't really like standalone Mystic Forge groups since they should be a general rule, and there's many entries with a single item, rendering it useless outside very specific cases (core weapons and core back items).